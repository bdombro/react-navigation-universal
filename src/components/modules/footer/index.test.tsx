import * as React from "react";
import {shallow} from 'enzyme';
import {TestProvidersContainer} from "../../containers/TestProviders.container";
import {FooterSection} from "./";


describe('FooterModule', () => {

    it('should be hidden at mobile width', async () => {
        // TODO: Get shallow working. Has issue with paper.
        // const wrapper = shallow(<FooterSection/>, {wrappingComponent: TestProvidersContainer});
        // expect(wrapper).toMatchSnapshot();

        const wrapper = shallow(<TestProvidersContainer><FooterSection/></TestProvidersContainer>);
        console.dir(wrapper.html());
        expect(wrapper.html()).toMatchSnapshot();


    });
});
