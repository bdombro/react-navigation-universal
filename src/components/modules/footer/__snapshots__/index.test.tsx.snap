// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterModule should be hidden at mobile width 1`] = `"<View style=\\"flex:1\\"><View testID=\\"GlobalLayout\\" style=\\"flex:1;width:100%;overflow:hidden;background-color:hsl(219,23%,90%)\\"><View testID=\\"FooterSection\\" style=\\"align-items:center;background-color:hsl(219,23%,60%);padding-vertical:30px\\"><Text testID=\\"Text\\" style=\\"0:[object Object];1:[object Object]\\">Thar be footer!</Text></View></View></View>"`;
